be/src/vec/columns/column_object.h

```diff
@@ -222,6 +222,10 @@ class ColumnObject final : public COWHelper<IColumn, ColumnObject> {
     // this structure and fill with Subcolumns sub items
     mutable std::shared_ptr<rapidjson::Document> doc_structure;
 
+    // column with raw json strings
+    // used for quickly row store encoding
+    ColumnPtr original_column;
```

Comment:
original_column is not so accurate since VARIANT column can be used to store parquet, orc and other complex file instead JSON only. rowstore_column may be better.

Reply:
done